ALVRTrackingInterface/ALVRTrackingInterface.cs

```diff
@@ -0,0 +1,412 @@
+            
+            if (pitch_L > 0)
+            {
+                expressions[(int)FBExpression.Eyes_Look_Left_L] = Math.Min(1, (float)(pitch_L / eyeLookOutLimit));
+                expressions[(int)FBExpression.Eyes_Look_Right_L] = 0;
+            }
+            else
+            {
+                expressions[(int)FBExpression.Eyes_Look_Left_L] = 0;
+                expressions[(int)FBExpression.Eyes_Look_Right_L] = Math.Min(1, (float)((-pitch_L) / eyeLookInLimit));
+            }
+            if(yaw_L > 0)
+            {
+                expressions[(int)FBExpression.Eyes_Look_Up_L] = Math.Min(1, (float)(yaw_L / eyeLookUpLimit));
+                expressions[(int)FBExpression.Eyes_Look_Down_L] = 0;
+            }
+            else
+            {
+                expressions[(int)FBExpression.Eyes_Look_Up_L] = 0;
+                expressions[(int)FBExpression.Eyes_Look_Down_L] = Math.Min(1, (float)((-yaw_L) / eyeLookDownLimit));
+            }
+
+            if (pitch_R > 0)
+            {
+                expressions[(int)FBExpression.Eyes_Look_Left_R] = Math.Min(1, (float)(pitch_R / eyeLookInLimit));
+                expressions[(int)FBExpression.Eyes_Look_Right_R] = 0;
+            }
+            else
+            {
+                expressions[(int)FBExpression.Eyes_Look_Left_R] = 0;
+                expressions[(int)FBExpression.Eyes_Look_Right_R] = Math.Min(1, (float)((-pitch_R) / eyeLookOutLimit));
+            }
+            if (yaw_R > 0)
+            {
+                expressions[(int)FBExpression.Eyes_Look_Up_R] = Math.Min(1, (float)(yaw_R / eyeLookUpLimit));
+                expressions[(int)FBExpression.Eyes_Look_Down_R] = 0;
+            }
+            else
+            {
+                expressions[(int)FBExpression.Eyes_Look_Up_R] = 0;
+                expressions[(int)FBExpression.Eyes_Look_Down_R] = Math.Min(1, (float)((-yaw_R) / eyeLookDownLimit));
+            }
+
+            #endregion
+
+            #region Eye Data to UnifiedEye
+
+            //Porting of eye tracking parameters
+            eye.Left.Gaze = MakeEye
+            (
+                LookLeft: expressions[(int)FBExpression.Eyes_Look_Left_L],
+                LookRight: expressions[(int)FBExpression.Eyes_Look_Right_L],
+                LookUp: expressions[(int)FBExpression.Eyes_Look_Up_L],
+                LookDown: expressions[(int)FBExpression.Eyes_Look_Down_L]
+            );
+
+            eye.Right.Gaze = MakeEye
+            (
+                LookLeft: expressions[(int)FBExpression.Eyes_Look_Left_R],
+                LookRight: expressions[(int)FBExpression.Eyes_Look_Right_R],
+                LookUp: expressions[(int)FBExpression.Eyes_Look_Up_R],
+                LookDown: expressions[(int)FBExpression.Eyes_Look_Down_R]
+            );
+
+            // Eye dilation code, automated process maybe?
+            eye.Left.PupilDiameter_MM  = 0.0035f;
+            eye.Right.PupilDiameter_MM = 0.0035f;
```

Comment:
EyesDilation on legacy parameters is being driven to 0 with input from ALVR, users must disable eye dilation to go back to default. I believe the mutators zeroing out the pupil diameter with no input. Previous QuestPro controls would just hold default value of 0.73 on the float value.

Reply:
The .73 value you're talking about was controlling the old EyesDilation parameter directly (which just represented the value between 0-1), currently the value that is expected is the exact MM value from an interface, which then on runtime is used to create the old EyesDilation ranges. 

I can just have it so that this interface sets the dilation ranges from 0 - 10, and then the pupil diameter can be set to whatever it needs to be (which we would expect to be between 5 - 6 MM for normal pupil diameter ranges)

Reply:
See if https://github.com/benaclejames/VRCFaceTracking/pull/122/commits/74f3b62885b1ce48787ac666778226e703c7801b resolves your issue!